lib/cacheable/version.rb

```diff
@@ -3,14 +3,12 @@
 module Cacheable
   module VERSION
     MAJOR = 1
-    MINOR = 0
-    TINY = 4
+    MINOR = 1
+    TINY = 0
```

Comment:
We should probably do a major version bump if we are deprecating older versions of ruby.

Reply:
Sounds good esp because I realized that [endless ranges](https://github.com/splitwise/cacheable/pull/14/files#diff-28d861d812fd80349186f4ab77a9475024d05e23cf42385d4929664ba913f807R17) aren't available in pre-2.6